src/pages/RepoPage/SettingsTab/tabs/ConfigurationManager/components/FeatureItem/FeatureItem.tsx

```diff
@@ -0,0 +1,89 @@
+import A from 'ui/A'
+import Icon from 'ui/Icon'
+
+interface FeatureItemProps extends React.PropsWithChildren {
+  name: string
+  configured?: boolean
+  docsLink?: string // navLink key
+  getStartedLink: string // navLink key
+  hiddenStatus?: boolean
+}
+
+function FeatureItem({
+  name,
+  configured,
+  docsLink,
+  hiddenStatus,
+  getStartedLink,
+  children,
+}: FeatureItemProps) {
+  return (
+    <div>
```

Comment:
`l`: Do we need this outer wrapping div, or could we replace with a fragment `<> ... </>`?